### ISSUE TITLE
Avatar offset in membership updates in the TL

### ISSUE THREAD

**Issue by HarHarLinks** [vector-im/element-web#22744]
### Steps to reproduce

![image](https://user-images.githubusercontent.com/2803622/177494953-067e2fa1-25f5-4497-a297-1971a7a87694.png)

public accessible test case: https://matrix.to/#/!zzGPdcyFqxIOJEijoI:matrix.org/$WHpUKcDce0SzHUAzjQhHVblI1aFlu_Oo-ThzAIcg0PY?via=matrix.org&via=element.io&via=envs.net

Might apply to non-membership events that are rendered similarly, i.e. might share the css with them or not.

#### What did you expect?
vertical centering with enough padding/resizing to accomodate
#### What happened instead?
see picture

### Operating system

arch

### Application version

Element Nightly version: 2022070501 Olm version: 3.2.8

### How did you install the app?

aur

### Will you send logs?

No

**Comment by luixxiul** [comment]
I suppose it has been fixed with https://github.com/matrix-org/matrix-react-sdk/pull/8965 yesterday, if it is a compact modern layout.

**Comment by HarHarLinks** [comment]
Yes, my settings are Modern layout + compact layout + Default images.

Updated screenshot using Element Nightly version: 2022070601 Olm version: 3.2.8
![image](https://user-images.githubusercontent.com/2803622/177558250-d85fc25c-fd87-493c-ae27-053aaf19371a.png)

- avatar isn't perfectly vertically centered (looks like 0 top but 2 px bottom padding)
- timestamp has different alignment than either avatar or the event text

you judge whether that's as intended or not

**Comment by luixxiul** [comment]
IMO it cannot be fixed perfectly unless flexbox is applied there, which is a big task (which requires the refactoring of EventTile). Otherwise it would be required to modify position with a couple of pixel, though it is a stupid solution considering the fact that each language has different height of characters (mind Arabic languages, Chinese/Japanese/Korean languages, etc).
